examples/routing_service/routing_service_msql_adapter/cpp/rs_dll/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+#
+# (c) 2018 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+#  RTI grants Licensee a license to use, modify, compile, and create derivative
+#  works of the Software.  Licensee has the right to distribute object form
+#  only for use with RTI products.  The Software is provided "as is", with no
+#  warranty of any type, including any warranty for fitness for any purpose.
+#  RTI is under no obligation to maintain or support the Software.  RTI shall
+#  not be liable for any incidental or consequential damages arising out of the
+#  use or inability to use the software.
+
+
+cmake_minimum_required(VERSION 3.12)
+project (MsqlAdapter)
+
+if(DEFINED ENV{NDDSHOME})
+	# NDDSHOME is set
+else()
+    set(msg
+        "NDDSHOME environment variable must be set")
+    message(FATAL_ERROR ${msg})
+endif()
+
+if(NOT BUILD_SHARED_LIBS)
+    set(msg
+        "Static linking is not supported for this example. You can build it "
+        "using the -DBUILD_SHARED_LIBS=1 CMake parameter")
+    message(FATAL_ERROR ${msg})
+endif()
+
+if(NOT CONNEXTDDS_ARCH)
+    set(msg
+        "CONNEXTDDS_ARCH must be defined. For example, -DCONNEXTDDS_ARCH=x64Win64VS2017")
+    message(FATAL_ERROR ${msg})
+endif()
+
+# Find RTI Connext dependencies
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    "$ENV{NDDSHOME}/resource/cmake"
+)
+
+# Find RTI Connext dependencies
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CONNEXTDDS_DIR}/resource/cmake")
+find_package(
+    RTIConnextDDS "6.0.1"
+    REQUIRED
+    COMPONENTS
+        core
+        routing_service
+)
+
+# Compiler Definitions 
+add_compile_definitions(RTI_RS_USE_LOG) # if defined use logging
```

Comment:
Since this definition is associated to the `msqladapter` target, please, add it using the CMake `target_compile_definitions` macro: https://cmake.org/cmake/help/latest/command/target_compile_definitions.html

Reply:
Changed the README.md files and the CMakeLists.txt files. Will commit when all changes have been done.